Firestore/src/Query.php

```diff
@@ -852,6 +890,48 @@ private function newQuery(array $additionalConfig, $overrideTopLevelKeys = false
      */
     private function finalQueryPrepare(array $query)
     {
+        if ($this->limitToLast) {
```

Comment:
Just double checking that this will only ever be called once  - even if another clause is added after?

Reply:
Yep, this method is invoked when a client calls `Query::documents()`.